concourse/setup.py

```diff
@@ -98,30 +98,33 @@ def create_image_pull_secret(
 
 
 @ensure_annotations
-def create_tls_secret(namespace: str, config_path: str):
-    """Creates the TLS secret for concourse web component in the K8s cluster"""
-    # XXX: move those to cc-config
-    tls_key_file = "concourse-tls.key"
-    tls_cert_file = "concourse-tls.crt"
-    tls_secret_name = "concourse-web-tls"
-
-    secret_helper = KubernetesSecretHelper(kubeutil.Ctx().create_core_api())
+def create_tls_secret(
+    config_set: ConfigurationSet,
+    namespace: str,
+):
+    """Creates the configured TLS secret for the Concourse web-component in the K8s cluster"""
+    ensure_not_none(config_set)
+    ensure_not_empty(namespace)
+
+    # secret-name is predetermined by the Helm-chart
+    tls_secret_name = 'concourse-web-tls'
```

Comment:
at least make this an argument of this function; even better: mv it to concourse-cfg (in cc-config). pass-in from there. if possible with reasonable efforts, also pass to helm-deploy; if this is not (easily) possible, document this in cc-config